apps/todo-app-api/src/modules/todo-category/dto/response/todo-category.dto.ts

```diff
@@ -0,0 +1,14 @@
+import { ToDoCategoryTable } from 'todo-common-lib/mysql';
+
+export class TodoCategoryDto {
+  id?: number;
+  name?: string;
+  slug?: string;
+  color?: number;
+  updatedAt?: Date;
+  createdAt?: Date;
+
+  constructor(category?: ToDoCategoryTable) {
+    Object.assign(this, { ...category });
```

Comment:
これは型が台無しになるので避けた方が良いと思います。
意図しないプロパティが増えててバグになる原因になると思います。
面倒かもしれませんが1プロパティずつ代入した方が良いと思います。
(他の6箇所も同様)

```suggestion
    this.id = category?.id;
    this.name = category?.name;
    this.slug = category?.slug;
    this.color = category?.color;
    this.updatedAt = category?.updatedAt;
    this.createdAt = category?.createdAt;
```
    this.createdAt = category?.createdAt;
```